changelogs.py

```diff
@@ -0,0 +1,35 @@
+import re
+import subprocess
+import sys
+from datetime import date
+import fileinput
+
+# Save new changelogs in temp.md
+changelogs_cmd = "semantic-release changelog"
+output = subprocess.check_output(changelogs_cmd.split())
+output = "## [" + sys.argv[1] + "] - " + str(date.today()) + "\n"+output.decode("utf-8")
+
+with open('temp.md', 'w') as file:
+    file.writelines(output)
```

Comment:
`writelines` works here, but is confusing: writelines is documented to take a list of strings as an argument, but you are giving it a single string.  You should use `.write(output)` here.